ImageGallery/src/org/sleuthkit/autopsy/imagegallery/ImageGalleryTopComponent.java

```diff
@@ -88,6 +89,31 @@
     private VBox leftPane;
     private Scene myScene;
 
+    /**
+     * Returns whether the ImageGallery window is open or not.
+     * 
+     * @return true, if Image gallery is opened, false otherwise
+     */
+    @ThreadConfined(type = ThreadConfined.ThreadType.JFX)
+    public static boolean isImageGalleryOpen() {
+         
+        final TopComponent topComponent =  WindowManager.getDefault().findTopComponent(PREFERRED_ID);
+        if (topComponent != null) {
+            return topComponent.isOpened();
+        }
+        return false;   
+    }
+    
+    /**
+     * Returns the top component window.
+     * 
+     * @return Image gallery top component window, null if it's not open
+     */
+    @ThreadConfined(type = ThreadConfined.ThreadType.JFX)
+    public static TopComponent getTopComponent() {
+        return WindowManager.getDefault().findTopComponent(PREFERRED_ID);  
+    }
+    
```

Comment:
The annotation on these two methods are not correct. The methods are called in the EDT.